doc/DeveloperGuide.md

```diff
@@ -94,6 +94,10 @@ Use case ends.
 2. Should be able to hold up to 1000 persons.
 3. Should come with automated unit tests and open source code.
 4. Should favor DOS style commands over Unix-style commands.
+5. Should come with a help feature and various helper messages so that it is usable by a novice who had never had CLI experience before.
+6. The project is expected to have at least 4 additional major and minor features by the end of the semester.
```

Comment:
This is not an NFR as it does not relate to your product =P, this is more of a school requirement. School requirement != product NFR!